src/popup/router/pages/SignTransaction.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <!-- eslint-disable vue-i18n/no-raw-text -->
+  <div class="sign-transaction">
+    <div class="section-title">Sign transaction for</div>
+
+    <div class="url-bar link-sm text-left">
+      {{ callbackOrigin }}
+    </div>
+
+    <div class="section-title">Transaction</div>
+
+    <div class="tip-note-preview mt-15">
+      {{ $route.query.encodedTx }}
```

Comment:
This page can not have UI at all, you may just call `accounts/confirmTxSigning` from `mounted`